recipes/conda-store-split/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "conda-store" %}
+{% set version = "0.2.5" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/c/conda-store/conda_store-{{ version }}.tar.gz
+    sha256: bbfdfd0f8224e128147523edc0d44c400fdeb164e5053454c15b81db306843f2
+    folder: conda-store
+  - url: https://pypi.io/packages/source/c/conda-store-server/conda-store-server-{{ version }}.tar.gz
+    sha256: 4622e5e32ee7948db41ee1e9767bf8fc9bdc260a57665c85fb3c6110db35de48
+    folder: conda-store-server
+
+build:
+  number: {{ build }}
+
+outputs:
+  - name: "conda-store"
+    version: {{ version }}
+    build:
+      number: {{ build }}
+      noarch: python
+    script: "build_conda_store.sh"
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+        - jupyter-packaging
+      run:
+        - python >=3.6
+        - jupyterlab >=3
+      run_constrained:
+        - {{ pin_subpackage('conda-store-server', exact=True) }}
+
+    test:
+      imports:
+        - conda_store
+
+  - name: "conda-store-server"
+    version: {{ version }}
+    build:
+      number: {{ build }}
+      noarch: python
+    script: "build_conda_store_server.sh"
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - python >=3.6
+        - conda-docker
+        - conda-pack
+        - sqlalchemy
+        - requests
+        - flask
+        - flask-cors
+        - pyyaml
+        - pydantic
+        - minio
+        - typer
+      run_constrained:
+        - {{ pin_subpackage('conda-store', exact=True) }}
+
+    test:
+      requires:
+        - conda-store
```

Comment:
This probably won't fail as `run_exports` and `run_constrained` aren't properly applied inside a single recipe.

Reply:
I see. Should I remove it or just leave it?

Reply:
Turns out it results in a dependency conflict. I will debug locally once the noarch artifacts are enabled. #15223